servers/src/main/java/tachyon/master/file/FileSystemMaster.java

```diff
@@ -1326,8 +1335,21 @@ public void reportLostFile(long fileId) throws FileDoesNotExistException {
     return mDependencyMap.getPriorityDependencyList();
   }
 
+  /**
+   * Loads metadata for the object identified by the given path from UFS into Tachyon.
```

Comment:
In both cases the function creates an inode in the inode tree for representing the object. I am not sure what else to say besides describing the implementation details. Conceptually, there is no difference between loading a directory and loading a file.